src/test/scripts/product_versions.py

```diff
@@ -15,18 +16,25 @@ def get_lts_version(argv):
 		product = 'jira-software'
 	elif product == 'bitbucket':
 		product = 'stash'
-
 	if product in known_supported_version:
-		url = f"https://my.atlassian.com/download/feeds/archived/{product}.json"
+		url_archived = f"https://my.atlassian.com/download/feeds/archived/{product}.json"
+		url_current = f"https://my.atlassian.com/download/feeds/current/{product}.json"
 
 		try:
-			fl = urllib.request.urlopen(url)
-			fdata = fl.read()
-			jsdata = json.loads(fdata[10:len(fdata)-1].decode("utf-8"))
+			# load archived feeds
```

Comment:
Minor request can we be a bit more descriptive on the naming of the 4 variables below i.e.
`fdata_a`
`jsdata`
`fdata_c`
`jsdata`

Reply:
👍